src/bench/bench.go

```diff
@@ -128,9 +128,9 @@ func run_benchmark(ctx *cli.Context, name string, tasks int, functions int, func
 		waiting -= 1
 	}
 
-	if errors > (errors+successes)/100 {
-		panic(fmt.Sprintf(">1%% of requests failed (%d/%d)", errors, errors+successes))
-	}
+	// if errors > (errors+successes)/100 {
```

Comment:
Let's not get rid of this safety check.  You could always make the threshold configurable if you really want.